zipkin/src/main/scala/org/http4s/zipkin/algebras/CollectorOp.scala

```diff
@@ -0,0 +1,14 @@
+package org.http4s.zipkin.algebras
+
+import org.http4s.zipkin.models.ZipkinInfo
+
+import scalaz.Free
+
+// We might consider just making this a trait with an abstract `send` method.
+sealed trait CollectorOp[A]
+
+object CollectorOp {
+  final case class Send(zipkinInfo: ZipkinInfo) extends CollectorOp[Unit]
+  def send(zipkinInfo: ZipkinInfo): Collector[Unit] =
+    Free.liftFC(Send(zipkinInfo))
```

Comment:
I'm sure this'll come off as a stupid question, but how do I write it so that it's compatible with both? The 7.2 API is backward-compatible, I'm guessing?